content/pages/en/meetups.rst

```diff
@@ -17,16 +17,27 @@ Meetups
 Meetups
 ~~~~~~~
 
-We have monthly meetups with talks and workshop on various python related and development in general and data topics.
+In our monthly Meetup you can look forward to workshops and talks on various
+Python topics, e.g. from the field of software development or data science.
 
-You can find and join upcoming events on our `Meetup group <https://www.meetup.com/PyLadies-Berlin/>`__.
+In addition, we start our setup events with simple topics that will make your
+start with programming with Python easier.
 
-We do have varying levels in different meetups from novice to expert in specific topics. We always try to
-include a "Non-coding superpower" 10 mins talk in the beginning. Because we believe (tech) life is much more than coding!
+You can find and join upcoming events on our `Meetup group
+<https://www.meetup.com/PyLadies-Berlin/>`__.
+
+We do have varying levels in different meetups from novice to expert in
+specific topics. We always try to
```

Comment:
I would say, 'We often start our events with a 10 minute "Non-coding superpower" talk because ...'